### ISSUE TITLE
Does not work on stock 68000 CPU

### ISSUE THREAD

**Issue by Villadelfia** [paulroberthill/BlueSCSI-toolbox-Amiga#4]
The programs refuse to start on a stock 68000 CPU with error #80000004.

**Comment by paulroberthill** [comment]
I never thought a 68000 user would ever use this! I'll see about a 68000 build.
It requires Reaction. (OS 3.5 or OS 3.2.x)

**Comment by Villadelfia** [comment]
Yeah I'm using it on 3.2, I just tried rebuilding it for 68000 and it seems to work fine except for the logo being messed up, so I just removed that gadget from the UI.

**Comment by Towersen** [comment]
Hi, I´m using it on "stock" 68000 processor Commodore CDTV, with SCSI controller (Matthias Heinrichs design) with BlueSCSI V2 connected via 50 pin SCSI cable, and have 8 Mb Fast RAM under 68000 processor (GottaGo FastRAM 8MB) and Dual Kickstart (1.3 and 3.2) and Extended 2.35 CDTV ROMs. Everything is enough to run AGS system with dual boot to AGS Games System and AGS 3.1 Workbench. I have a second HD Image with AmigaOS 3.2. All works like a charm, not very fast of corse, but all fine. But have the same problem, toolbox refuse to start with an error in both system. The Commodore CDTV has a physical CD drive, but it's damaged, so I need to configure the CD images using BlueSCSI and be able to exchange them with a toolbox. Thanks in advance!

**Comment by paulroberthill** [comment]
I've been trying to add support for older versions of the OS but it looks bad.
I'll see what I can do this weekend.

I see there are some additions to the BlueSCSI API which could be interesting.

**Comment by Towersen** [comment]
But it didn't work for me on Workbench 3.1 or Amiga OS 3.2, and the latter isn't an old system. I think the problem must lie in the use of the stock 68000 processor, the 8Mb RAM expansion, or something with the CDTV. I don´t know...

**Comment by Villadelfia** [comment]
As I said in my earlier message, it just needs to be compiled with the flag to only use base 68000 instructions. Nothing in this program benefits from 68020+ instructions anyway.

**Comment by paulroberthill** [comment]
There's a new release with some fixes from Stefan Reinauer. This includes a 68000 build. Please see if this helps.

**Comment by Villadelfia** [comment]
Sadly my Amiga with the bluescsi died, and I went with a pistorm in my other Amiga. But I'm confident it works, it worked when I made my own build, I just had to remove the image from the gui.
